dle/cfn/deploy_django.yml

```diff
@@ -0,0 +1,285 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  ElasticIPAddress:
+    Type: String
+    Default: "44.197.203.81"
+    Description: IP address of EC2 instance where Elastic-kibana are located
+
+Resources:
+  CloudMap:
+    Properties:
+      Description: Service Map for Docker Compose project dle
+      Name: dle.local
+      Vpc: vpc-23074059
+    Type: AWS::ServiceDiscovery::PrivateDnsNamespace
+  Cluster:
+    Properties:
+      ClusterName: dle
+      Tags:
+      - Key: com.docker.compose.project
+        Value: dle
+    Type: AWS::ECS::Cluster
+  Default8000Ingress:
```

Comment:
Should only have HTTP / HTTPS open to the world, not `8000`. SSH but with IP restrictions since we don't have a VPN peered with VPC. Check out the `entrypoint` script, if you set `USE_NGINX` it will use `nginx` + `gunicorn` and run on `:80` instead. We could also change the default behavior to do that.